Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -1392,6 +1392,33 @@ void SoftwareGLContext::gl_get_floatv(GLenum pname, GLfloat* params)
     }
 }
 
+void SoftwareGLContext::gl_color_mask(GLboolean red, GLboolean green, GLboolean blue, GLboolean alpha)
+{
+    u32 mask = m_rasterizer.color_mask();
```

Comment:
Please use `m_rasterizer.options()` and `m_rasterizer.set_options()` instead. This pattern is already used in different parts of the library.